metrics/collectors/images/images_collector.py

```diff
@@ -92,6 +93,7 @@ def collect(self):
                 # the path ends with the image filename
                 image_name = path_table[-1]
 
+                image_type = "preinstalled" if ".img.xz" in path else "live"
```

Comment:
the "type" info has to come from the path, see the code above.

- "daily-preinstalled" should also have flavor=ubuntu
- "daily-live" -> type = live
- "daily-preinstalled" => type = preinstalled
- by default: type = None